projects/browser-extension/vue-app/src/components/app-shell/app-shell.vue

```diff
@@ -34,7 +34,8 @@ import RefreshButton from './refresh-button';
 export default {
   components: { AppSidebar, ExternalLink, PanelSection, RefreshButton },
   computed: {
-    ...mapState([ 'head' ]),
+    // @TODO :: Clean this up
```

Comment:
Are you making a ticket for all the TODO's?

Reply:
Yes ☺️